test/Generator/TypesTest.php

```diff
@@ -113,10 +117,10 @@ public function setTypeProvider()
         $range_float = 1 << (PHP_INT_SIZE === 8 ? 53 : 24);
 
         $values = [
-            ['smallint', 01,                                                    ],
-            ['smallint', 0x1,                                                   ],
-            ['integer',  01,                                                    ],
-            ['integer',  0x1,                                                   ],
```

Comment:
Why remove the trailing commas here?

Reply:
This is a single-line array, which doesn't have trailing comma's inside of it, since they are unnecessary comma's and pointless.

`[1,2,],` is incorrect, since it should be `[1,2],`.